### DISCUSSION TITLE
An additional Nightscout-related alert

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2846] [Ideas]
We have one alert related to a Nightscout uploader.
It alerts when upload fails.  But, a lot of times one may be away from internet and that alert will trigger then frequently.  

I would like to add one more alert that will only alert if the total number readings stuck in the queue reaches a certain limit, for example 288, which is the number of readings xDrip uploads in 24 hours.  And I will add a parameter to it that the user can choose to change the number of hours.

So, you can choose to have an alert if xDrip cannot upload for any of the following.
2 hours
8 hours
24 hours
48 hours
36 hours  

Is this a bad idea?

**Comment by tzachi-dar** [comment]
This seems like a good idea to me.

**Comment by Navid200** [comment]
The existing alert triggers if there are no uploads for 30 minutes.
So, instead of adding another alert, we can just add the ability to edit the length of time of failure required to trigger the alert.

**Reply by ElyeM** [reply]
That's actually a good idea, a few weeks ago FreeDns was down for several hours, and afterwards my nightscout was down. I just disabled the cloud upload entirely because of the alerts.

**Reply by Navid200** [reply]
I have opened a PR: https://github.com/NightscoutFoundation/xDrip/pull/2858
It changes the alert frequency from once every half hour to once a day.

**Comment by Navid200** [comment]
Change of plans:

There will be one log per 6 hours.
There will be one notification per day.
No new setting.

https://github.com/NightscoutFoundation/xDrip/pull/2858
